jib-gradle-plugin/src/integration-test/resources/gradle/projects/simple/build-java11-incompatible.gradle

```diff
@@ -0,0 +1,18 @@
+plugins {
+  id 'java'
+  id 'com.google.cloud.tools.jib'
+}
+
+sourceCompatibility = 11
+targetCompatibility = 11
+
+repositories {
+  mavenCentral()
+}
+
+dependencies {
+  compile files('libs/dependency-1.0.0.jar')
+}
+
+jib.from.image = 'gcr.io/distroless/java'
```

Comment:
Should we qualify this with the `8` tag? (In case we change `latest` to be Java 11 in the future)

Reply:
My idea was to leave the tag out so that if `latest` becomes Java 11, then builds will break and we will be immediately notified. When the switch happens, we will fix the test by qualifying this with the `8` tag, giving the chance to update other places that need attention.